openshift/templates/nsp/build.yaml

```diff
@@ -0,0 +1,65 @@
+---
+kind: Template
+apiVersion: v1
+metadata:
+  name: global-nsp-build-template
+objects:
+  - kind: NetworkSecurityPolicy
+    apiVersion: security.devops.gov.bc.ca/v1alpha1
+    metadata:
+      name: pods-to-k8s-api
+      labels:
+        name: pods-to-k8s-api
+        env: ${ENV_NAME}
+    spec:
+      description: |
+        Allow pods to talk to the internal k8s api so builds work.
+        This only needs to be specified once per environment.
+      source:
+        - - $namespace=${NAMESPACE_NAME}-${ENV_NAME}
+      destination:
+        - - int:network=internal-cluster-api-endpoint
+
+  - kind: ExternalNetwork
+    apiVersion: security.devops.gov.bc.ca/v1alpha1
+    metadata:
+      name: internet-build-resources
+      network: internet-build-resources
+      labels:
+        name: internet-build-resources
+        network: internet-build-resources
+    spec:
+      description: |
+        Define the network parameters for accessing https resources on internet.
+      entries:
+        - 0.0.0.0/0
+      servicePorts:
+        - tcp/443
+        - tcp/80
+
+  - kind: NetworkSecurityPolicy
+    apiVersion: security.devops.gov.bc.ca/v1alpha1
+    metadata:
+      name: pods-to-external-network
+      labels:
+        name: pods-to-external-network
+        env: ${ENV_NAME}
+    spec:
+      description: |
+        Allow the builds to access the internet.
+        This only needs to be specified once per environment.
+      source:
+        - - $namespace=${NAMESPACE_NAME}-${ENV_NAME}
+      destination:
+        - - ext:name=internet-build-resources
```

Comment:
OUtadet, needs to be updated with new KNPs

Reply:
updated nsp